docs/_developers/dns-discovery-setup.md

```diff
@@ -0,0 +1,113 @@
+---
+title: DNS Discovery Setup Guide
+sort_key: C
+---
+
+This document explains how to set up an [EIP 1459][dns-eip] node list using the cmd/devp2p
+developer tool. The focus of this guide is creating a public list for the Ethereum mainnet
+and public testnets, but you may also find this helpful if you want to set up DNS-based
+discovery for a private network.
+
+DNS-based node lists can serve as a fallback option when connectivity to the discovery DHT
+is unavailable. In this guide, we'll create node lists by crawling the discovery DHT, then
+publishing the resulting node sets under chosen DNS names.
+
+### Installing the devp2p command
+
+cmd/devp2p is a developer utility and is not included in the Geth distribution. You can
```

Comment:
Oh, I see, it's not part of Geth. Why? I mean you may want to write a sentence on the reasoning behind it.

Reply:
I don't want to explain why cmd/devp2p functionality is not in geth in this document. It's a developer tool for peer-to-peer networking tasks.